yang/frr-zebra.yang

```diff
@@ -1845,6 +1905,38 @@ module frr-zebra {
           "Link bandwidth informational parameter, in megabits.";
       }
       // TODO -- link-params for (experimental/partial TE use in IGP extensions)
+      container state {
+        config false;
+        description
+          "Operational data.";
+        leaf zif-type {
+          type identityref {
+            base zebra-interface-type;
+          }
+        }
+
+        leaf vlan-id {
+          type uint16 {
+            range "1..4094";
+          }
+        }
+
+        leaf vni-id {
```

Comment:
vni-id read only state?

Reply:
This is under zebra's `state` container in read-only fields. Similar to the one fetched from zebra's `show interface` command output.